pep-0695.rst

```diff
@@ -1151,6 +1149,37 @@ upper and lower bounds on a type.
     // Alternate form of generic function
     function func2(v::Container{T} where T <: Real)
 
+Dart
+-----
+
+Dart uses angle brackets to declare type parameters and for specialization.
+The upper bound of a type is specified using the "extends" keyword. By default,
+type parameters are covariant.
+
+Dart supports declaration-site variance where variance of type parameters is
+explicitly declared using "in", "out" and "inout" keywords. It does not support
+use-site variance.
+
+Dart provides no way to specify a default type argument.
+
+::
```

Comment:
```suggestion
.. code-block:: dart
```

I'm pretty sure all the code blocks are mixing the appropriate syntax highlighting lexer, meaning they get lexed and highlighted as Python instead, so they do not display as intended. Either @KotlinIsland or @erictraut , do you think you could add the appropriate [Pygments lexer short name](https://pygments.org/docs/lexers/) to each one to resolve this? Thanks!

<details>

<summary>Examples</summary>

![image](https://user-images.githubusercontent.com/17051931/180450942-7cc57697-beca-46af-8f46-d84da149996c.png)

![image](https://user-images.githubusercontent.com/17051931/180451047-7de626dc-84f5-4cbd-a1e5-af9267aaae93.png)

![image](https://user-images.githubusercontent.com/17051931/180451113-0cfd70fb-ffe8-4cdf-98c4-a5d7fdcd0eb5.png)


</details>